### PR TITLE
Adds plots in SampleData Overview

### LINKED ISSUE #2834: The Sample Data example should include all the timeseries data
Currently the [sample data overview](http://docs.sunpy.org/en/stable/generated/gallery/acquiring_data/2011_06_07_sampledata_overview.html) does not show plots of all the timeseries sample data.

So the sample data overview page has examples of GOES, RHESSI and we would like to add examples of NORH,NOAA for example.

### PR DESCRIPTION
**Pull request by Krish2208**
## PR Description

Adds NORH, NOAA(Overplot of SRS_TABLE over HMI Image), EVE, LYRA, GBM Plots to the Sample Data Overview as per the issue, Fixes #2834 

## PR Checklist

- [x] I have followed the guidelines in the [Contributing document](https://docs.sunpy.org/en/latest/dev_guide/contents/newcomers.html)
- [x] Changes follow the coding style of this project
- [x] Changes have been formatted and linted
- [ ] Changes pass pytest style unit tests (and `pytest` passes). [N/A]
- [ ] Changes include any required corresponding changes to the documentation
- [x] Changes have a descriptive commit message with a short title
- [x] Changes have been rebased and squashed to the minimal number (typically 1) of relevant commits
- [ ] I have added a `Fixes #XXXX -` or `Closes #XXXX -` comment to auto-close the issue that your PR addresses

### CODE REVIEW

**Comment by Krish2208** [comment]
pre-commit.ci autofix

**Comment by nabobalis** [comment]
Rendered version: https://sunpy--6015.org.readthedocs.build/en/6015/generated/gallery/acquiring_data/2011_06_07_sampledata_overview.html#sphx-glr-generated-gallery-acquiring-data-2011-06-07-sampledata-overview-py

File: examples/acquiring_data/2011_06_07_sampledata_overview.py [hunk 1]
```diff
@@ -94,3 +98,67 @@
...
+eve = sunpy.timeseries.TimeSeries(sample_data.EVE_TIMESERIES, source='EVE')
+fig = plt.figure()
+eve.plot()
+plt.show()
+
+####################################################################################
+# LYRA
+
+lyra = sunpy.timeseries.TimeSeries(sample_data.LYRA_LEVEL3_TIMESERIES, source='LYRA')
+fig = plt.figure()
```
Comment (nabobalis) [inline examples/acquiring_data/2011_06_07_sampledata_overview.py:154]: The final plot is kind of small, could you make it larger?

File: examples/acquiring_data/2011_06_07_sampledata_overview.py [hunk 2]
```diff
@@ -94,3 +98,67 @@
...
+lyra.plot()
+plt.show()
+
+####################################################################################
+# GBM
+
+gbm = sunpy.timeseries.TimeSeries(sample_data.GBM_TIMESERIES, source='GBMSummary')
+fig = plt.figure()
+gbm.plot()
```
Comment (nabobalis) [inline examples/acquiring_data/2011_06_07_sampledata_overview.py:163]: The X axis labels overlap, I would to fix it.

**Comment by Krish2208** [comment]
Ok, I will fix these issues.

File: examples/acquiring_data/2011_06_07_sampledata_overview.py [hunk 3]
```diff
@@ -94,3 +98,67 @@
...
+                fontweight='bold')
+
+plt.show()
+
+####################################################################################
+# EVE
+
+eve = sunpy.timeseries.TimeSeries(sample_data.EVE_TIMESERIES, source='EVE')
+fig = plt.figure()
+eve.plot()
```
Comment (nabobalis) [inline examples/acquiring_data/2011_06_07_sampledata_overview.py:147]: The time axis label is cut off, I think the figure needs to be smaller?

File: examples/acquiring_data/2011_06_07_sampledata_overview.py [hunk 4]
```diff
@@ -94,3 +98,67 @@
...
+##############################################################################
+# NORH
+
+norh = sunpy.timeseries.TimeSeries(sample_data.NORH_TIMESERIES)
+fig = plt.figure()
+norh.plot()
+plt.show()
+
+####################################################################################
+# NOAA
```
Comment (nabobalis) [inline examples/acquiring_data/2011_06_07_sampledata_overview.py:111]: We should metion its overlaid with HMI.

**Review by nabobalis — APPROVED** [review]

**Comment by nabobalis** [comment]
Thanks for the PR @Krish2208

**Pull request merged**